Glyssen/Controls/ScriptBlocksViewer.cs

```diff
@@ -163,45 +163,54 @@ private void HandleDataGridViewBlocksCellValueNeeded(object sender, DataGridView
 				e.Value = m_getDelivery(block);
 			}
 		}
+
+		private void HideToolTip()
+		{
+			if(m_toolTip != null)
+			{
+				m_toolTip.Hide(this);
+				m_toolTip.Dispose();
+				m_toolTip = null;
+			}
+		}
 		#endregion
 
 		#region Browser events
 		private void OnMouseOver(object sender, DomMouseEventArgs e)
 		{
 			if (e.Target == null || m_getCharacterIdForUi == null || !m_blocksDisplayBrowser.Visible)
+			{
+				HideToolTip();
 				return;
+			}
 
 			var geckoElement = e.Target.CastToGeckoElement();
-			var divElement = geckoElement as GeckoDivElement;
-			if (divElement == null)
-				return;
+			var checkElement = geckoElement as GeckoHtmlElement;
 
-			if (divElement.Parent.ClassName == BlockNavigatorViewModel.kCssClassContext)
+			for (int i = 0; i < 10; i++)
 			{
-				m_toolTip = new ToolTip {IsBalloon = true};
+				if(checkElement == null)
```

Comment:
I know I'm nitpicking here, but "normal" C# style is no space after method names but spaces after for, if, etc.
Please fix this one and the one on else if below.